src/version/delete-version.ts

```diff
@@ -1,7 +1,7 @@
 import {from, Observable, merge, throwError, of} from 'rxjs'
-import {graphql} from '@octokit/graphql'
 import {catchError, map, tap} from 'rxjs/operators'
 import {GraphQlQueryResponse} from '@octokit/graphql/dist-types/types'
+import {graphql} from './graphql'
```

Comment:
To enable mocking, i switched to importing a local file instead. The problem is `@actions/github` exports a class which introduces a challenge for mocking... solved by adding one more level of indirection :)